components/whyd/2022/Terminal/UserInputArea.vue

```diff
@@ -141,31 +142,36 @@ export default {
     this.terminalLinesQueue.reverse()
 
     const paths = new Set(
-      Object.values(this.terminalCommands).map((command) => command.path)
+      Object.values(this.terminalCommands).map((command) => command.path),
     )
     this.validPaths = paths
 
     // Valid commands
-
     const prettyNames = Array.from(Object.values(this.terminalCommands)).map(
-      (e) => e.prettyName
+      (e) => e.prettyName,
     )
 
     this.validCommands = new Set(
-      prettyNames.concat(Array.from(Object.keys(this.terminalCommands)))
+      prettyNames.concat(Array.from(Object.keys(this.terminalCommands))),
     )
 
-    // Retrive the data
+    this.usedCommands =
+      JSON.parse(localStorage.getItem('whyd22.usedCommands')) ?? []
+    for (const command of this.usedCommands) {
+      if (this.terminalCommands[command] != null)
+        this.terminalCommands[command].hasBeenRun = true
+    }
```

Comment:
IDK which section this should go in, but it'll get moved if/when this function is refactored anyway